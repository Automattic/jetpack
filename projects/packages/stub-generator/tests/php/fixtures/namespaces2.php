<?php

return array(
	'files' => array(
		'files/functions.php' => array(
			'class' => '*',
		),
		'files/functions_ns.php' => '*',
	),
);
