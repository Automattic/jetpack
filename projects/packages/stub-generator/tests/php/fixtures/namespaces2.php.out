<?php
namespace Some\NS;

/**
 * A namespaced function.
 *
 * @param string $p1 Parameter 1.
 * @param array $p2 Parameter 2.
 * @return ?object Some object.
 */
function a_namespaced_function($p1, $p2)
{
}
